### PR TITLE
docs: APIM recommendations are PG verified  now and updates for stv1 retirement

### PR DESCRIPTION
**Pull request by aarthiem**
# Overview/Summary
This PR includes updates to stv1 retirement date and all apim recommendations are PG verified now, so the flag is set to true.

### Breaking Changes

None

## As part of this pull request I have

- [x ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [ x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [x ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [ x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by oZakari** [comment]
Hi @aarthiem, thanks for the PR, can you please take a look at the failed tests? Looks like the length of the description for one of your recommendations is too long and the PR title needs to be adjusted.

**Comment by aarthiem** [comment]
@oZakari - Fixed the long description, please review.

File: azure-resources/ApiManagement/service/recommendations.yaml [hunk 1]
```diff
@@ -46,9 +46,9 @@
   recommendationResourceType: Microsoft.ApiManagement/service
   recommendationMetadataState: Active
   longDescription: |
-    Upgrading to API Management stv2 is required as stv1 retires on 31 Aug 2024, offering enhanced capabilities with the new platform version.
+    stv1 instances are deprecated on 31 Aug 2024 and if not migrated to stv2, auto migration will be triggered. In few cases, due to technical feasibility, services may be shut down in March 2025.
```
Comment (oZakari) [inline azure-resources/ApiManagement/service/recommendations.yaml:49]:
```suggestion
    stv1 instances will be deprecated on August 31, 2024. If not migrated to stv2 by then, auto-migration will occur. In some cases, due to technical limitations, services may be shut down in March 2025.
```

**Review by oZakari — APPROVED** [review]
LGTM

**Review by oZakari — DISMISSED** [review]

**Review by oZakari — DISMISSED** [review]

**Review by rodrigosantosms — APPROVED** [review]
approved

**Review by oZakari — APPROVED** [review]
LGTM

**Pull request merged**